internal/engine/wazevo/backend/regalloc/regalloc_test.go

```diff
@@ -335,17 +340,21 @@ func TestAllocator_livenessAnalysis(t *testing.T) {
 					continue
 				}
 				t.Run(fmt.Sprintf("block_id=%d", blockID), func(t *testing.T) {
-					exp := tc.exp[blockID]
+					exp := tc.exps[blockID]
 					if len(exp.liveIns) == 0 {
 						require.Nil(t, actual.liveIns, "live ins")
 					} else {
-						sort.Slice(actual.liveIns, func(i, j int) bool {
-							return actual.liveIns[i] < actual.liveIns[j]
+						var actuals []VRegID
+						for _, s := range actual.liveIns {
+							actuals = append(actuals, s.v.ID())
+						}
+						sort.Slice(actuals, func(i, j int) bool {
+							return actuals[i] < actuals[j]
```

Comment:
Now that 1.23 is out, we could bump the minimum version to 1.21 and start using `slices.Sort` for this code.

Reply:
good catch, but it's not yet released right (still rc1?)